src/plugins/lists/base/hooks/api.tsx

```diff
@@ -35,6 +35,7 @@ export interface Card<T extends Entry> {
 interface EntryFields {
   label: string;
   name: string;
+  type: string;
```

Comment:
This can just be optional string
```suggestion
  type?: string;
```